control/autoware_control_command_gate/src/command/subscription.cpp

```diff
@@ -25,17 +25,18 @@ CommandSubscription::CommandSubscription(uint16_t id, const std::string & name,
   using std::placeholders::_1;
   const auto control_qos = rclcpp::QoS(5);
   const auto volatile_qos = rclcpp::QoS(1);
+  const auto transient_qos = rclcpp::QoS(1).transient_local();
   const auto prefix = "~/inputs/" + name + "/";
 
   sub_control_ = node.create_subscription<Control>(
     prefix + "control", control_qos, std::bind(&CommandSubscription::on_control, this, _1));
   sub_gear_ = node.create_subscription<GearCommand>(
     prefix + "gear", volatile_qos, std::bind(&CommandSubscription::on_gear, this, _1));
   sub_turn_indicators_ = node.create_subscription<TurnIndicatorsCommand>(
-    prefix + "turn_indicators", volatile_qos,
+    prefix + "turn_indicators", transient_qos,
```

Comment:
Did you check there is no volatile publishers for turn_indicators and hazard_lights. 
As you know, if the publisher is volatile, transient subscriber only get the latest topic.